django_tenants/management/commands/__init__.py

```diff
@@ -95,13 +95,26 @@ def get_tenant_from_options_or_interactive(self, **options):
             while True:
                 tenant_schema = input("Enter Tenant Schema ('?' to list schemas): ")
                 if tenant_schema == '?':
-                    print('\n'.join(["%s - %s" % (t.schema_name, t.get_primary_domain()) for t in all_tenants]))
+                    print('\n'.join(["%i) %s - %s" % (i, t.schema_name, t.get_primary_domain()) for i, t in enumerate(all_tenants, start=1)]))
                 else:
                     break
 
-        if tenant_schema not in [t.schema_name for t in all_tenants]:
+        tenant_schema_index = 0
+        tenant_schema_index_is_valid = False
+
+        try:
+            tenant_schema_index = int(tenant_schema) - 1
+        except ValueError:
+            pass
+        else:
+            tenant_schema_index_is_valid = 0 < tenant_schema_index <= len(all_tenants) 
+
+        if tenant_schema not in [t.schema_name for t in all_tenants] and not tenant_schema_index_is_valid:
             raise CommandError("Invalid tenant schema, '%s'" % (tenant_schema,))
 
+        if tenant_schema_index_is_valid:
+            return all_tenants[tenant_schema_index]
```

Comment:
I'm quite certain what this line is supposed to do in contrast with L118. Looks like we could just return the tenant model without querying it again.